Document-Processing/introduction.md

```diff
@@ -45,31 +45,31 @@ Description
 Essential<sup>&reg;</sup> PDF
 </td>
 <td>
-.NET Class library used to create, read, and write PDF files in [.NET Framework](https://www.syncfusion.com/document-processing/pdf-framework/net/) [Windows Forms, WPF, ASP.NET MVC, ASP.NET], [.NET Core](https://www.syncfusion.com/document-processing/pdf-framework/net-core/), [UWP](https://www.syncfusion.com/document-processing/pdf-framework/uwp/), [Xamarin](https://www.syncfusion.com/document-processing/pdf-framework/xamarin/), [WinUI](https://www.syncfusion.com/document-processing/pdf-framework/winui/) and [.Net MAUI](https://www.syncfusion.com/document-processing/pdf-framework/maui/) applications.
+<a href="https://www.syncfusion.com/document-sdk/net-pdf-library">.NET class library</a>  used to create, read, and write PDF files in .NET Framework [Windows Forms, WPF, ASP.NET MVC], .NET Core, UWP, WinUI, .NET MAUI, and Blazor applications.
 </td>
 </tr>
 <tr>
 <td>
 Essential<sup>&reg;</sup> XlsIO
 </td>
 <td>
-.NET Class library used to create, read, edit, write, and convert Microsoft Excel files in [.NET Framework](https://www.syncfusion.com/document-processing/excel-framework/net/) [Windows Forms, WPF, ASP.NET MVC, ASP.NET], [.NET Core](https://www.syncfusion.com/document-processing/excel-framework/net-core/), [UWP](https://www.syncfusion.com/document-processing/excel-framework/uwp/), [Xamarin](https://www.syncfusion.com/document-processing/excel-framework/xamarin/), [WinUI](https://www.syncfusion.com/document-processing/excel-framework/winui/) and [.Net MAUI](https://www.syncfusion.com/document-processing/excel-framework/maui/) applications.
+<a href="https://www.syncfusion.com/document-sdk/net-excel-library">.NET class library</a>  used to create, read, and write PDF files in .NET Framework [Windows Forms, WPF, ASP.NET MVC], .NET Core, UWP, WinUI, .NET MAUI, and Blazor applications.
```

Comment:
Content is incorrect. provide xlsio content instead of PDF

Reply:
Updated